### ISSUE TITLE
Issue with Ethereum Wallet and mist do not open

### ISSUE THREAD

**Issue by dominic-n** [ethereum/mist#3399]
- [x] I've asked for help in the [Mist Gitter](http://gitter.im/ethereum/mist) before filing this issue.

```
Version: `0.0.0`
OS & Version: linux
Node version: `geth 0.0.0` 
Number of blocks synchronized: 0
```
I have installed mist and ethereum wallet but they fail when opened saying "Ethereum node connection failure" 
ScreenShot:
![d39041a6-09da-11e7-9762-b7b2c43c9a0d](https://user-images.githubusercontent.com/16276190/33794233-99f9cbac-dcbf-11e7-8a0c-e239d24f9d4b.png)
I have seen a similar issue but it was closed due to inactivity but not answered since he was asked for log files but didnt provide them. Here are the log files any help will be appreciated, I have been stuck for almost a day and I feel there is something I havent installed but do not know what.
Mist log files:
[node.log](https://github.com/ethereum/mist/files/1544507/node.log)
Ethereum wallet log file:
[node.log](https://github.com/ethereum/mist/files/1544508/node.log)

**Comment by evertonfraga** [comment]
Hi @dominic-n, sorry for letting you without answers on the other issue.

Thanks for sending the log files. From there I can spot the issue:
```
WARN [12-09|11:24:20] System clock seems off by 2h59m55.770843641s, which can prevent network connectivity
WARN [12-09|11:24:20] Please enable network time synchronisation in system settings.```

It seems your clock is not synced. Would you try syncing your clock with NTP servers (or automatically time-syncing)?

**Comment by jacksenechal** [comment]
This may be related to https://github.com/ethereum/mist/issues/3461

**Comment by evertonfraga** [comment]
@dominic-n I suppose after updating your clock it should work. Either way, https://github.com/fjl/os-timesync/pull/9 will solve that problem.

**Comment by evertonfraga** [comment]
Fixed in 0.10.0.
